### DISCUSSION TITLE
[Feature] Hide top bar on clicking an image.

### DISCUSSION THREAD

**Discussion by SayantanRC** [immich-app/immich#14474] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Immich web always overlays the top bar on top of images. Although it is transparent, the icons are visible and distracting.  

<img src=https://github.com/user-attachments/assets/f8cebbde-1735-40d9-bdbe-e0f4455edf5a height=400/>  

Kindly consider allowing hiding the top bar in some way. One way would be if the user clicks on the image, the top bar is hidden.  

This is a continuation of this closed issue: https://github.com/immich-app/immich/issues/14471

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#10083
